src/app.js

```diff
@@ -1 +1,28 @@
 'use strict';
+
+const args = process.argv.slice(2);
+
+let name = 'John';
+let lastname = 'Doe';
+let age = 30;
+
+args.forEach(arg => {
+  const [paramName, paramValue] = arg.split('=');
+
+  switch (paramName) {
+    case '--name':
+      name = paramValue;
+      break;
+    case '--lastname':
+      lastname = paramValue;
+      break;
+    case '--age':
+      age = paramValue;
+      break;
+    default:
+      break;
+  }
+});
```

Comment:
You can really simplify your solution by using [minimist](https://www.npmjs.com/package/minimist) module. Try to use it